raddb/sites-available/default

```diff
@@ -1139,20 +1132,16 @@ send Access-Accept {
 	#  work, you will need to define the WiMAX NAI, usually
 	#  via:
 	#
-#	update request {
-#		&WiMAX-MN-NAI = "%{User-Name}"
-#	}
+#	&request.WiMAX-MN-NAI = "%{User-Name}"
 
 	#  If you want various keys to be calculated, you will
 	#  need to update the reply with "template" values. The
 	#  module will see this, and replace the template values
 	#  with the correct ones taken from the cryptographic
 	#  calculations, e.g.
 	#
-#	update reply {
-#		&Vendor-Specific.WiMAX.FA-RK-Key = 0x00
-#		&Vendor-Specific.WiMAX.MSK = "%{reply.EAP-MSK}"
-#	}
+#	&reply.Vendor-Specific.WiMAX.FA-RK-Key = 0x00
```

Comment:
I'm not sure that the RADIUS encoder will encode nested attributes for vendor-specific.  We'll have to test / verify that before merging this PR.

i.e. `src/tests/unit/protocols/radius/wimax.txt` doesn't contain tests for nested WiMAX attributes, such as:

```
Vendor-Specific = { WiMAX = { Capability = { Release = "1.0" } } }
```

We'd have to add tests for that before changing the `Vendor-Specific` attributes to no longer use `update`.

Reply:
for sure, I took all notes about that issue. thanks for the explanation.